src/Android/MainActivity.cs

```diff
@@ -22,25 +22,8 @@
 
 namespace Bit.Droid
 {
-    [Activity(
-        Label = "Bitwarden",
-        Icon = "@mipmap/ic_launcher",
-        Theme = "@style/LaunchTheme",
-        MainLauncher = true,
-        LaunchMode = LaunchMode.SingleTask,
-        ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation |
-                               ConfigChanges.Keyboard | ConfigChanges.KeyboardHidden |
-                               ConfigChanges.Navigation | ConfigChanges.UiMode)]
-    [IntentFilter(
-        new[] { Intent.ActionSend },
-        Categories = new[] { Intent.CategoryDefault },
-        DataMimeTypes = new[]
-        {
-            @"application/*",
-            @"image/*",
-            @"video/*",
-            @"text/*"
-        })]
+    // Activity and IntentFilter declarations have been moved to Properties/AndroidManifest.xml
+    // They have been hardcoded so we can use the default LaunchMode on Android 12+
```

Comment:
Can you add a note about making sure any future changes are made in both manifest files?  (I can see forgetting about the `values-v31` version)